domains/xenapp.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "techsideofficial",
+        "email": "wienerwolf77@gmail.com"
+    },
+    "record": {
+        "TXT": "e8ervtqg_bno4xnob40f7rgseijrvp72jkhd-siwl0o"
```

Comment:
TXT record alone will not do anything.

Reply:
I can't change it
<img width="417" alt="image" src="https://github.com/is-a-dev/register/assets/68258126/51214a1c-cffd-416b-b4bf-562d834203c4">